app/handlers/signup_start.rb

```diff
@@ -23,7 +23,8 @@ def handle
     end
 
     # If email in use, want users to login with that email, not create another account
-    fatal_error(code: :email_in_use) if email_in_use?
+    return outputs[:status] = :email_in_use if email_in_use?
```

Comment:
ah, in this case you can just use the `fatal_error` -- that'll take you to the `failure` block in the `handle_with` call.  In there, you can do different things based on `@handler_result.errors`.  Output statuses are good, but typically I like to still split them into the `success` and `failure` camps.